### PR TITLE
DevExp RN additions

### PR DESCRIPTION
**Pull request by codyhoag**
Preview: http://file.rdu.redhat.com/~choag/041620/rn-4.4-updates-4/release_notes/ocp-4-4-release-notes.html#ocp-4-4-developer-experience

### CODE REVIEW

**Comment by codyhoag** [comment]
@adambkaplan this PR adds release notes for several of your DEVEXP comments. Can you review this to make sure I got everything right? Thanks!

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -170,6 +170,31 @@ The {product-version} tested maximum for the number of Pods per node is 500.
...
 Limit Calculator] to estimate cluster limits for your environment.
 
+[id="ocp-4-4-developer-experience"]
+=== Developer experience
+
+[id="ocp-4-4-automatic-image-pruning"]
+==== Automatic image pruning
+
+Image pruning is now performed automatically. This is managed by the Registry
+Operator, which creates a CronJob to run periodic image pruning.
```
Comment (adambkaplan) [inline release_notes/ocp-4-4-release-notes.adoc:180]: Late breaking issue - cluster admins will need to enable this feature in 4.4.

See https://bugzilla.redhat.com/show_bug.cgi?id=1825284

File: release_notes/ocp-4-4-release-notes.adoc [hunk 2]
```diff
@@ -170,6 +170,31 @@ The {product-version} tested maximum for the number of Pods per node is 500.
...
+You can also use commands like `oc wait` to view the timestamp of a build state
+transition.
+
+[id="ocp-4-4-recreate-rollouts-for-image-registry"]
+==== Recreate rollouts for image registry
+
+You can now use the `Recreate` rollout strategy when deploying the image
+registry. This lets you use `ReadWriteOnce` persistent volumes to successfully
+upgrade an {product-title} cluster. This also allows the image registry to use
+block storage types like Elastic Block Store on Amazon Web Services.
```
Comment (adambkaplan) [inline release_notes/ocp-4-4-release-notes.adoc:196]:
```suggestion
registry. This lets you use `ReadWriteOnce` persistent volumes, such as Elastic Block Storage on AWS.
When using such storage types, the `Recreate` rollout strategy must be used to successfully upgrade an {product-title} cluster.
```

File: release_notes/ocp-4-4-release-notes.adoc [hunk 3]
```diff
@@ -170,6 +170,31 @@ The {product-version} tested maximum for the number of Pods per node is 500.
...
+Image pruning is now performed automatically. This is managed by the Registry
+Operator, which creates a CronJob to run periodic image pruning.
+
+[id="ocp-4-4-build-objects-report-conditions-in-status"]
+==== Build objects report conditions in status
+
+Build conditions have been added for each existing {product-title} build phase.
+These conditions contain information about the build during its build lifecycle.
+You can also use commands like `oc wait` to view the timestamp of a build state
+transition.
```
Comment (adambkaplan) [inline release_notes/ocp-4-4-release-notes.adoc:188]:
```suggestion
You can also use commands like `oc wait` to wait for a specific build phase to be reached.
```

**Review by adambkaplan — CHANGES_REQUESTED** [review]
@codyhoag a few nits.

**Comment by codyhoag** [comment]
@adambkaplan thanks for your feedback. Can you take a second look?

File: release_notes/ocp-4-4-release-notes.adoc [hunk 4]
```diff
@@ -235,6 +235,30 @@ Limit Calculator] to estimate cluster limits for your environment.
 [id="ocp-4-4-developer-experience"]
 === Developer experience
 
+[id="ocp-4-4-automatic-image-pruning"]
+==== Automatic image pruning
+
+You can now enable automatic image pruning. This is not available by default;
```
Comment (adambkaplan) [inline release_notes/ocp-4-4-release-notes.adoc:241]:
```suggestion
You can now enable automatic image pruning. This is not turned on by default;
```

File: release_notes/ocp-4-4-release-notes.adoc [hunk 5]
```diff
@@ -235,6 +235,30 @@ Limit Calculator] to estimate cluster limits for your environment.
...
+you will be notified of this option after installing or upgrading to
+{product-title} 4.4. This automation is managed by the Registry Operator, which
```
Comment (adambkaplan) [inline release_notes/ocp-4-4-release-notes.adoc:243]:
```suggestion
{product-title} 4.4. This automation is managed by the Image Registry Operator, which
```

**Review by adambkaplan — CHANGES_REQUESTED** [review]
Minor nits.

**Comment by codyhoag** [comment]
@adambkaplan I made those additional adjustments. Let me know if there's anything else. Thanks!

**Review by adambkaplan — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-4-release-notes.adoc [hunk 6]
```diff
@@ -235,6 +235,30 @@ Limit Calculator] to estimate cluster limits for your environment.
...
+Build conditions have been added for each existing {product-title} build phase.
+These conditions contain information about the build during its build lifecycle.
+You can also use commands like `oc wait` to wait for a specific build phase to
+be reached.
+
+[id="ocp-4-4-recreate-rollouts-for-image-registry"]
+==== Recreate rollouts for image registry
+
+You can now use the `Recreate` rollout strategy when deploying the image
+registry. This lets you use `ReadWriteOnce` persistent volumes, such as Elastic
```
Comment (bobfuru) [inline release_notes/ocp-4-4-release-notes.adoc:258]: s/such as Elastic Block Storage on AWS/such as AWS Elastic Block Store/
(This is how we reference it [elsewhere](https://docs.openshift.com/container-platform/4.3/storage/persistent_storage/persistent-storage-aws.html) in the docs.)

**Comment by bobfuru** [comment]
One small nit and then LGTM.

**Pull request merged**